internal/cli/get.go

```diff
@@ -61,7 +62,8 @@ func NewCmdGet() *cobra.Command {
 	cmd.Flags().StringVarP(&o.Output, "output", "o", o.Output, fmt.Sprintf("Output format. One of: (%s).", strings.Join(legalOutputTypes, ", ")))
 	cmd.Flags().Int32Var(&o.Limit, "limit", o.Limit, "The maximum number of results returned in the list response.")
 	cmd.Flags().StringVar(&o.Continue, "continue", o.Continue, "Query more results starting from the value of the 'continue' field in the previous response.")
-	cmd.Flags().StringVarP(&o.FleetName, "fleetname", "f", o.FleetName, "Fleet name for accessing individual templateversions.")
+	cmd.Flags().StringVar(&o.FleetName, "fleetname", o.FleetName, "Fleet name for accessing templateversions (templateversions only).")
```

Comment:
minor: Could we clarify the description a little around what  `templateversions only` means.